qmxgraph/html.py

```diff
@@ -1,22 +1,108 @@
-def create_image_tag_str(source, width, height):
+import attr
+
+
+asdict = attr.asdict
+
+
+@attr.s(repr=False, slots=True, hash=True)
+class _HtmlChildTypeValidator(object):
```

Comment:
I think it would be simpler to make a simple function and bind `child_types` using `functools.partial`